addons/l10n_br/models/account.py

```diff
@@ -4,12 +4,20 @@
 from odoo import fields, models
 
 
+class AccountTaxGroup(models.Model):
+    _inherit = 'account.tax.group'
+
+    tax_discount = fields.Boolean(string='Discount this Tax in Prince',
```

Comment:
`in Prince` ➡️ `from the Price`. This whole sentence deserves better phrasing in English (and in AccountTaxTemplate as well.